README.md

```diff
@@ -1,77 +1,129 @@
+To run this application first you must clone the repo.
 
-```
-MYSQL_HOST=localhost
-MYSQL_PORT=3306
-MYSQL_DATABASE=cytube3
-MYSQL_USER=cytube3
-MYSQL_PASSWORD=UltraSecretPass
-MYSQL_ROOT_PASSWORD=UltraSecretRootPass
-HTTP_PORT=80
-HTTP=true
-HTTPS_PORT=443
-HTTPS=false
-IO=true
-IO_PORT=1337
-IO_DOMAIN=http://localhost
-ROOT_DOMAIN=localhost
-USE_MINIFY=false
-COOKIE_SECRET=change-me
-SYNC_CRTKEY=null
-SYNC_CRT=null
-SYNC_CRTCA=null
-SYNC_TITLE=Sync
-SYNC_DESCRIPTION="Free, open source synchtube"
-YOUTUBE_KEY=null
-CHANNEL_STORAGE=file
-VIMEO_WORKAROUND=false
-TWITCH_ID=null
-MIXER_ID=null
-GIT_COMMIT=0bc866dbfaba0ef09dc703d6ef261bdf2e70b2ff
-```
+Next you will need to run get_latest_configs.sh in order to automatically pull the latest versions of the config from the sync repo. You may also choose to do this manually. You will need the conf/example and a config.yaml files so that they can be mounted into the container. The script is there to make this easier for you by downloading the necessary files.
 
-If you will be using the local mysql instance and are using it for production, you will want to specify a bind mount to ensure that your user and channel information is retained across deployments. The Alpine-based MariaDB image is a drop-in replacement for the official MariaDB image, so you can reference their page for further documentation:
+After you have the files, open the config.yaml file and edit the information in the config to meet your specific needs.
+calzoneman has thoroughly commented the file to give you a good idea of what needs to be edited.
 
-https://hub.docker.com/_/mariadb/
+After editting your config.yaml file you will need to edit the docker-compose.yml to match what you have entered in the config.yaml file. It is commented to give you an idea of what needs to be editted.
 
-This will soon be updated with the option to not install and run MariaDB, which will be the most ideal for those using this for production with a remote sql server. A cluster environment is to follow, likely with Compose and/or Kubernetes, as well as a dev tag/branch.
+When you have finished the previous steps start the containers by running ```docker-compose up -d``` this will cause docker-compose to build the Dockerfile and start the sync and mariadb/mysql containers.
 
-Build:
+## docker-compose.yml
 
+This is the docker-compose.yml file, in this file you will need to update the environmental variables, ports, paths to your certificate files (if using SSL) and the path to your config file. Failure to update this will cause sync/cytube to not start.
+The file has comments to help you edit what is necessary.
 ```
-docker build -t cytube-docker .
+version: "3.7"
```

Comment:
Is this the same file that is also saved as `docker-compose.yml` in this pull request?  If so I think we can just keep that commented one and remove the one from README.md for clarity (and to avoid them getting out of sync)

Reply:
Removed this from the readme for clarity.